src/reactions/ReactionListItem.js

```diff
@@ -0,0 +1,42 @@
+/* @flow */
+import React, { PureComponent } from 'react';
+import { View } from 'react-native';
+
+import type { AggregatedReaction, NamedUser } from '../types';
+import { Label, RawLabel, LineSeparator } from '../common';
+import Emoji from '../emoji/Emoji';
+
+type Props = {
+  aggregatedReaction: AggregatedReaction,
+};
+
+export default class ReactionListItem extends PureComponent<Props> {
+  props: Props;
+
+  static contextTypes = {
+    styles: () => null,
+  };
+
+  render() {
+    const { styles } = this.context;
+    const { name, users } = this.props.aggregatedReaction;
+    const usersNames = users.map((user: NamedUser) => user.full_name).join(', ');
+
+    return (
+      <View style={[styles.reactionListItemWrapper]}>
+        <View style={[styles.row]}>
+          <Emoji name={name} size={30} />
+          <View style={[styles.halfMarginLeft, styles.flexed]}>
```

Comment:
Do you need two `Views` inside each other here?
Also, add more space around the icon, twice what it is (so not 'half' :) )